base/partr.jl

```diff
@@ -208,6 +204,8 @@ function multiq_deletemin()
         end
     end
 
+    @assert @isdefined(rn1) "Assertion to tell the compiler about the definedness of this variable"
```

Comment:
Should this be a comment instead of a string?

Reply:
Maybe, but I'm not sure what to put for the assertion string. Any idea?